Change Core Philosophy (#755)
vtex/faststore: README.md

  - [An open ecommerce framework](#an-open-ecommerce-framework)
  - [Why building FastStore?](#why-building-faststore)

>   - [Project Philosophy](#project-philosophy)

Author: - [Project Philosophy](#project-philosophy)
- [Basic Architecture](#basic-architecture)

>   - [Why Jamstack?](#why-jamstack-and-gatsby)

Author: ```suggestion
  - [Why Jamstack?](#why-jamstack)
```

- Jamstack based (deploy with Netlify, Vercel and any other JAMStack platform)
- Marketplace ready with support to millions of skus
- Intelligence via Google Analytics and GTM

> - Evergreen stores with auto updates

Reviewer: How about "Evergreen stores that are always updated" or even "Evergreen stores that are automatically updated"?

> - Evergreen stores with auto updates

Author: ```suggestion
- Evergreen stores
```

1. Performance - Great score on Lighthouse and Core Web Vitals
2. Stability - E2E tests are complete and make sure the critical paths are covered
3. SEO - Stores should pass on all aspects of Google search console, like complete structured data and web vitals
4. Feature completion - Enable you to craft your custom ecommerce experience

> ## Basic Architecture

Reviewer 2: We don't have this section in table of contents/summary.

FastStore core is composed of two main packages:
1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library

> 2. `store-sdk`: An sdk for ecommerce to solve your basic ecommerce needs, like cart, pixel managment, session etc.

Reviewer 3: ```suggestion
2. `store-sdk`: An SDK for ecommerce to solve your basic ecommerce needs, like cart, pixel management, session etc.
```

> These packages have bidings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.

Reviewer 3: ```suggestion
These packages have bindings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
```

> We want to support other frameworks, like Next.JS and Create React App in the future

Reviewer: I'm not sure about listing Create React App as a framework. I would separate Next.JS from it. Something along the lines of: "We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App". 
Or not even mentioning Create React App at all 🤔  What do you think?

> We want to support other frameworks, like Next.JS and Create React App in the future

Reviewer 4: @tlgimenes What do you think about creating a section with future features or something like that and put this phrase there?

> We want to support other frameworks, like Next.JS and Create React App in the future

Author: ```suggestion
We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App"
```

> We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App"

Reviewer: ```suggestion
We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App
```
